section6Arrays/reverseString/main.go

```diff
@@ -0,0 +1,10 @@
+package rev_str
```

Comment:
use revstr as the package just like the library package strconv

Reply:
fixed